CRM/Contribute/Form/UpdateSubscription.php

```diff
@@ -74,40 +83,37 @@ public function preProcess() {
       CRM_Core_Error::statusBounce(ts('Required information missing.'));
     }
 
-    if ($this->_subscriptionDetails->membership_id && $this->_subscriptionDetails->auto_renew) {
-      // Add Membership details to form
-      $membership = civicrm_api3('Membership', 'get', [
-        'contribution_recur_id' => $this->contributionRecurID,
-      ]);
-      if (!empty($membership['count'])) {
-        $membershipDetails = reset($membership['values']);
-        $values['membership_id'] = $membershipDetails['id'];
-        $values['membership_name'] = $membershipDetails['membership_name'];
-      }
-      $this->assign('recurMembership', $values);
-      $this->assign('contactId', $this->_subscriptionDetails->contact_id);
+    $this->contributionRecur = \Civi\Api4\ContributionRecur::get(FALSE)
```

Comment:
Just noting that with what is merged now we don't need to / shouldn't do this because we already have that loaded & accessible via `$this->getContributionRecurValue()`